src/tss2-fapi/api/Fapi_Encrypt.c

```diff
@@ -339,6 +339,10 @@ Fapi_Encrypt_Finish(
         statecase(context->state, DATA_ENCRYPT_WAIT_FOR_RSA_ENCRYPTION);
             r = Esys_RSA_Encrypt_Finish(context->esys, &tpmCipherText);
             return_try_again(r);
+            if (r == 0x00000084) {
+                LOG_ERROR("Maybe the data to be encrypted is too large. Common values are "
```

Comment:
I would change it to "The data to be encrypted might be too large...."

Reply:
ACK